### PR TITLE
Npm publish

### PR DESCRIPTION
**Pull request by assiotis**
Setup so we can publish to npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,19 @@
 {
   "name": "chcli",
-  "version": "1.0.0",
+  "version": "1.0.2",
   "description": "A command line utility to make create clubhouse stories easier",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "publishConfig": {
+    "registry": "http://npm.adrise.tv"
```
Comment (springuper) [inline package.json:7]: should we do this in a public repo? i image we should publish this package to official npm registry.

File: package.json [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+  },
+  "preferGlobal": true,
+  "bin": {
```
Comment (springuper) [inline package.json]: suggestion: we should add a `files` property, most of package users don't need to install the ts files.
https://docs.npmjs.com/files/package.json#files.

**Review by springuper — APPROVED** [review]
LGTM.

leave small comments.

File: package.json (hunk 1 above)
Comment (assiotis) [reply package.json:7]: Yes, eventually I will

**Pull request merged**